packages/now-build-utils/src/detectors/angular.ts

```diff
@@ -1,14 +1,18 @@
 import { DetectorParameters, DetectorResult } from '../types';
 
 export default async function detectAngular({
-  fs: { hasDependency },
+  fs: { hasDependency, getPackageJsonBuildCommand, getDependencyVersion },
 }: DetectorParameters): Promise<DetectorResult> {
   const hasAngular = await hasDependency('@angular/cli');
   if (!hasAngular) return false;
   return {
-    buildCommand: 'ng build',
-    buildDirectory: 'dist',
+    buildCommand: (await getPackageJsonBuildCommand()) || 'ng build',
+    outputDirectory: 'dist',
     devCommand: 'ng serve --port $PORT',
```

Comment:
Similarly, I think we should extract the `dev` script if specified:

```suggestion
    devCommand: await getPackageJsonScript('dev', 'ng serve --port $PORT'),
```

Reply:
But perhaps not, since most of the time they won't be including the `$PORT` part 🤔

Reply:
Maybe we'd only prefer a `now-dev` script being defined, which we would document needs the `$PORT`…

Reply:
I'd leave the dev script out for now, since it's also common to have `"dev": "now dev"`, which wouldn't play well with this.